src/main/java/org/apache/commons/dbcp2/BasicDataSourceMXBean.java

```diff
@@ -131,6 +131,13 @@ default String getDefaultSchema() {
      */
     boolean isPoolPreparedStatements();
 
+    /**
+     * See {@link BasicDataSource#isClearStatementPoolOnReturn()}
+     * 
+     * @return {@link BasicDataSource#isClearStatementPoolOnReturn()}
+     */
+    boolean isClearStatementPoolOnReturn();
```

Comment:
You must use a default method here or this will break binary compatibility. 
Please add a Javadoc `since` tag for any new public or protected methods.

Reply:
The next version (for `since`) will be 2.8?

Reply:
Not quite, the POM shows `2.8.0-SNAPSHOT` so the next version will be `2.8.0`.